crates/sui-core/src/authority_aggregator.rs

```diff
@@ -298,6 +298,36 @@ impl<A> AuthorityAggregator<A> {
         }
     }
 
+    pub fn new_with_metrics(
```

Comment:
feel free to ignore changes in this file. It's part of https://github.com/MystenLabs/sui/pull/7023